app/Services/MovieServiceImpl.php

```diff
@@ -51,13 +55,46 @@ public function count(){
         return Movie::count();
     }
     public function create($movie){
+        $image = $movie['image'];
+        $savedImage = null;
+        if ($image){
+            $image_name = time() . '.' . $image->getClientOriginalExtension();
+
+            $destinationFullSize = public_path('storage/fullSize').'/'.$image_name;
+            $destinationThumbnail = public_path('storage/thumbnail').'/'.$image_name;
```

Comment:
umjesto ovog-a stavi `stroage_path` pa vidi sta ce ti vracati, trebaalo bi da ti vrati `storage/`, jer ovo sto si ti koristio je public putanja, to ti je ono kad odradis onaj `storage:link`, pa ti preveze file-ove iz storage-a u public folder. Trenutno se potrefilo da ti je putanja okej, ali u nekom slucaju ce putanje totalno drugacije da budu struktuirane